### PR TITLE
Run capybara under cuprite, as phantomjs is no longer in development

### PR DESCRIPTION
**Pull request by anderscarling**

### CODE REVIEW

File: godmin.gemspec
Comment (dgmstuart) [inline godmin.gemspec:37]: Should we also remove poltergeist from the dependencies?

**Review by dgmstuart — APPROVED** [review]
LGTM, apart from poltergeist still being a dev dependency?

File: godmin.gemspec
Comment (anderscarling) [reply godmin.gemspec:37]: My bad, will fix.

File: godmin.gemspec
Comment (anderscarling) [inline godmin.gemspec:40]:
```suggestion
```

**Review by dgmstuart — APPROVED** [review]
Still good - no need to request a re-review after an approved review, unless the changes resulting from PR feedback are particularly unusual or controversial.

**Pull request merged**
